mock-web-services/app.py

```diff
@@ -0,0 +1,31 @@
+from flask import Flask, Response, request, json
+from esri import EsriMockWebService
+
+app = Flask(__name__)
+esriWS = EsriMockWebService(app)
+
+
+@app.route("/esri/findAddressCandidates")
```

Comment:
**suggestion (blocking):** Please add a doc comment for this file and a README.md for this new directory